update_available_ios/lib/adapters/get_ios_package_version_impl.dart

```diff
@@ -4,11 +4,18 @@ import 'package:pub_semver/pub_semver.dart';
 import '../domain/get_ios_package_version.dart';
 import 'json.dart';
 
-const _lookupURL = 'https://itunes.apple.com/lookup';
+const _itunesURL = 'https://itunes.apple.com';
 
 GetIOSPackageVersion httpGetIOSPackageVersion() {
-  return (String bundleId) async {
-    final uri = Uri.parse('$_lookupURL?bundleId=$bundleId');
+  return (String bundleId, {String? appStoreRegion}) async {
+    Uri uri;
+
+    if (appStoreRegion == null) {
+      uri = Uri.parse('$_itunesURL/$appStoreRegion/lookup?bundleId=$bundleId');
+    } else {
+      uri = Uri.parse('$_itunesURL/lookup?bundleId=$bundleId');
+    }
```

Comment:
It seems you missed the order here:

```suggestion
    if (appStoreRegion == null) {
      uri = Uri.parse('$_itunesURL/lookup?bundleId=$bundleId');
    } else {
      uri = Uri.parse('$_itunesURL/$appStoreRegion/lookup?bundleId=$bundleId');
    }
```